### DISCUSSION TITLE
Can this be a Colab Notebook?

### DISCUSSION THREAD

**Discussion by astro9ine** [nulqwy/kohya_ss_colab#2] [General]
Can this be a Colab Notebook?

**Comment by nulqwy** [comment]
but it is a colab notebook?

**Reply by astro9ine** [reply]
where can i access colab netbook

**Comment by nulqwy** [comment]
> where can i access colab netbook

[link](https://colab.research.google.com/github/Spaceginner/kohya_ss_colab/blob/master/gui.ipynb) is literally in the sidebar?

![image](https://github.com/Spaceginner/kohya_ss_colab/assets/97038786/314b8b86-d6cc-4013-875c-97d16fcba1b1)
